packages/twenty-server/src/utils/getCompanyDomainName.ts

```diff
@@ -0,0 +1,8 @@
+// temporary, to remove once domainName has been fully migrated to Links type
+export const getCompanyDomainName = (company: any) => {
```

Comment:
Seems this one is not aligned with getCompanyDomainName from front package

Reply:
right, thanks